tests/fr/light_HassTurnOff.yaml

```diff
@@ -1,6 +1,20 @@
 language: fr
 tests:
   - sentences:
+      - Éteindre les lumières du salon
+      - Éteins la lumière du salon
+      - Éteins les lumières dans le salon
+      - Éteins toutes les lumières dans le salon
+      - Désactiver la lumière du salon
+    intent:
+      name: HassTurnOff
+      slots:
+        area: salon
+        domain: light
+    response: Lumières éteintes
+
+  - sentences:
```

Comment:
```suggestion
```
We don't need to duplicate sentences.

Reply:
Ok 
make sense, i have removed unnecessary sentence test